cmd/gitserver/server/server_grpc.go

```diff
@@ -34,6 +35,19 @@ func (gs *GRPCServer) Exec(req *proto.ExecRequest, ss proto.GitserverService_Exe
 		})
 	})
 
+	// Log which actor is accessing the repo.
+	args := req.GetArgs()
+	cmd := ""
+	if len(req.GetArgs()) > 0 {
+		cmd = req.GetArgs()[0]
+		args = args[1:]
+	}
```

Comment:
you already have a variable here:
```suggestion
	if len(args) > 0 {
		cmd = args[0]
		args = args[1:]
	}
```